### PR TITLE
3_navbar

### PR DESCRIPTION
**Pull request by AnaScia**
I added navbar with the names of the views and the title.

image

### CODE REVIEW

**Comment by AnaScia** [comment]
I added react-router with their respective links

image

File: src/App.jsx [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
+import Navbar from './component/Navbar';
+import { Routes, Route } from 'react-router-dom';
+import Budget from './view/Budget';
+import Blog from './view/Blog';
 
 function App() {
   return (
     <>
-      <Packing></Packing>
+      <Navbar />
```
Comment (Alvacampos) [inline src/App.jsx:8]: Good use of old react router, but I recommend you use the new version with outlet. This in a new way that Remix.run uses. 

It uses the component Outlet as a sort of conditional rendering, showing only the path to the view that you call. Here is a basic example

```
// Here you can declare a father route with its childrens, in the father route you will place both the navbar (that you want to show all the time) and the Outlet. The Outlet will show the component/view that you declared in children
const router = createBrowserRouter([
  {
    path: '/',
    element: <App />,
    errorElement: <div>Not Found</div>,
    children: [
      {
        path: '/',
        element: <Packing />,
      },
      {
        path: '/budget',
        element: <Budget />,
      },
      {
        path: '/blog',
        element: <Blog />,
      },
    ],
  },
]);

Comment (Alvacampos) [reply src/App.jsx:8]: All this goes in main, and in app you can have the navbar and the outlet

File: src/styles/navbar.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  background-image: url('../image/fondo_nubes.jpg');
+  align-items: center;
+}
+
+.navbar-container-div {
+  display: flex;
+  align-items: center;
+}
+.navbar-container-name {
+  margin: 0px 30px;
```
Comment (Alvacampos) [inline src/styles/navbar.css:13]: No need to use px when the value is 0

**Review by Alvacampos — COMMENTED** [review]
Very nice, just a few comments.

**Pull request merged**